### ISSUE TITLE
Docker Desktop: Document Mingw/msys2 POSIX path conversion in troubleshooting section

### ISSUE THREAD

**Issue by thaJeztah** [docker/docker.github.io#10912]
### Problem description

Users on Windows that use Mingw/msys2 for their shell may be running in the problem that that shell does "POSIX path conversion", see https://github.com/docker/for-win/issues/6754#issuecomment-629702199

Given that this "feature" can be confusing  and  lead to "surprising" issues, I think we should have a mention of this behaviour (and workarounds) in our documentation (probably under the "troubleshooting" section for Docker Desktop for Windows).

### Problem location

- I saw a problem on the following URL: <URL>

- I couldn't find the information I wanted. I expected to find it near the following URL: <URL>

- Other: <DETAILS>

**Comment by thaJeztah** [comment]
@stephen-turner @usha-mandya @StefanScherer (I kept forgetting to open an issue for this)

**Comment by thaJeztah** [comment]
FWIW; I could use some help on this with the right terminology / description. I'm not really sure in what cases/situations people are using mingw/msys2.

**Comment by StefanScherer** [comment]
Hello, Mac user here 👋, 
but happy to try to understand Windows use-cases.

What I learned is that [Git for Windows](https://gitforwindows.org/) brings a Git Bash which seems to be one popular way to do Git and some Linuxish commands on Windows in a pre-WSL world. I don't know if that matches msys2, as there is a standalone https://www.msys2.org/ installer as well. 🤔 
I went through these some years ago before I switched over to a MacBook Pro.

WSL 1 and now WSL 2 is coming to more and more users and I hope that will "fix" the situation on Windows, but my guess is that we will live for some years with all the spectrum of shells and tools.

**Comment by thaJeztah** [comment]
Yes, perhaps people should use alternatives. I mostly opened this because _when_ you hit this issue, it's really "unexpected" (why is it using these weird paths? I didn't specify them!), so if we somehow could point people to the options/workarounds, I think that's a good thing
